src/gui/widgets/FxLine.cpp

```diff
@@ -157,7 +166,13 @@ void FxLine::drawFxLine( QPainter* p, const FxLine *fxLine, bool isActive, bool
 	int width = fxLine->rect().width();
 	int height = fxLine->rect().height();
 
-	p->fillRect( fxLine->rect(), isActive ? fxLine->backgroundActive() : p->background() );
+	QColor color = muted ? ( isActive ? fxLine->backgroundActive().color() : p->background().color() )
+					: ( Engine::fxMixer()->effectChannel( m_channelIndex )->m_hasColor ? ( isActive ? Engine::fxMixer()->effectChannel( m_channelIndex )->m_color.darker( 120 ) 
+					: Engine::fxMixer()->effectChannel( m_channelIndex )->m_color.darker( 150 ) )
+					: ( isActive ? fxLine->backgroundActive().color()
+					: p->background().color() ) );
+
+	p->fillRect( fxLine->rect(), color );
```

Comment:
Ok this is one big mess, can rewrite or reformat to make it look better (though the logic functions)

Reply:
Yeah, I can't read this 😅. You could probably break it into two functions, like activeColor and inactiveColor.

Reply:
Okay, now that I substituted `Engine::fxMixer()->effectChannel( m_channelIndex )` with just `channel`, it should be a little more pleasing to the eye. If it's still confusing, I can change it.